File: app/src/main/java/com/kickstarter/ui/viewholders/EmptyViewHolder.java
package com.kickstarter.ui.viewholders;

import android.view.View;

import androidx.annotation.Nullable;

public final class EmptyViewHolder extends KSViewHolder {
  public EmptyViewHolder(final View view) {
    super(view);
  }

  @Override
  public void bindData(final @Nullable Object data) throws Exception {
  }

  @Override
  public void onBind() {
  }
}

File: app/src/main/java/com/kickstarter/ui/viewholders/EmptyViewHolder.kt
package com.kickstarter.ui.viewholders

import androidx.viewbinding.ViewBinding

class EmptyViewHolder(binding: ViewBinding) : KSViewHolder(binding.root) {
    @Throws(Exception::class)
    override fun bindData(data: Any?) {
    }

    override fun onBind() {}
}
